plugins/other/wordpress-multisite

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env sh
+# wordpress-multisite plugin
+#
+# Version 0.3
+# Date 2017-01-12
+# Some last minor fixes
+#
+# Version 0.2
+# Date 2016-10-24
+# Code improvements
+#
+# Version 0.1
+# Date 2016-02-07
+# Initial release
+#
+: <<=cut
+=head1 NAME
+
+Wordpress-Multisite Munin Plugin
+
+A Munin plugin to monitor posts, comments and pingbacks from every multisite instance.
+It uses multigraphs and also shows the combined number of posts, comments, pingbacks, instances and users.
+
+Most database queries came from the wordpress-mu-plugin which was written by Andre Darafarin and Chris Bair
+
+=head1 CONFIGURATION
+
+The plugin need access to the wordpress database
+
+=head2 Config file
+
+Create the config file plugin-conf.d/wordpress with the following values:
+
+  [wordpress*]
+  env.mysqlopts		# i.e. -uroot -prootpass
+  env.mysqlconnection	# defaults to -hlocalhost
+  env.database		# i.e. -Dwordpress
+  env.dbprefix		# defaults to wp_
+
+=head1 VERSION
+
+Version 0.3 (2017-01-12)
+
+=head2 Some minor fixes:
+
+  * fixed perldoc
+  * fixed some syntax warnings from shellcheck
+  * replaced expr by $(( ))
+
+=head1 AUTHOR
+
+Jonas Palm <jonaspalm . posteo . de>
+
+=head1 LICENSE
+
+GPLv3 or higher
+=cut
+
+# fill vars
+DB_OPTIONS=${mysqlopts}
+DB_CONNECTION=${mysqlconnection:--hlocalhost}
+DB_NAME=${database}
+DB_PREFIX=${dbprefix:-wp_}
+
+MYSQL_CMD=$(which mysql)
+wp_get() {
+	case $1 in
```

Comment:
Please add `local QUERY` (or at least expand the `case` with a default branch) in order to avoid hard-to-debug situations.